common/src/main/java/org/apache/atlas/utils/KafkaUtils.java

```diff
@@ -318,20 +318,9 @@ static String surroundWithQuotes(String optionVal) {
             return optionVal;
         }
 
-        String ret = optionVal;
-
-        // For property values which have special chars like "@" or "/", we need to enclose it in
-        // double quotes, so that Kafka can parse it
-        // If the property is already enclosed in double quotes, then do nothing.
-        if (optionVal.indexOf(0) != '"' && optionVal.indexOf(optionVal.length() - 1) != '"') {
```

Comment:
Probably, the `charAt` method could be used instead of `indexOf` , but I think we should omit this check at all, since the option value may contain double quotes at the beginning/at the end or in the middle of the string.
Thus, the following option values look valid to me:
```
user1
"user2
user3"
user"4
"user5" (the actual value is "user5", not user5)
```